README.md

```diff
@@ -9,7 +109,7 @@ It is fully free and fully open source. The license is Apache 2.0, meaning you a
 Logstash provides infrastructure to automatically generate documentation for this plugin. We use the asciidoc format to write documentation so any comments in the source code will be first converted into asciidoc and then into html. All plugin documentation are placed under one [central location](http://www.elasticsearch.org/guide/en/logstash/current/).
 
 - For formatting code or config example, you can use the asciidoc `[source,ruby]` directive
-- For more asciidoc formatting tips, see the excellent reference here https://github.com/elasticsearch/docs#asciidoc-guide
+- For more asciidoc formatting tips, see the excellent reference here https://github.com/elastic/docs#asciidoc-guide
```

Comment:
Thanks you for noticing, files in https://github.com/logstash-plugins/logstash-filter-example and https://github.com/elastic/logstash/tree/master/docs are the ones that need to be changes .... I will do a clean up later, otherwise feel free to do a PR there if you wish :+1: